src/Hl7.Fhir.Specification.Tests/Source/TerminologyTests.cs

```diff
@@ -160,88 +160,6 @@ public async T.Task TestPropertyRetrieval()
             Assert.True(conceptQuestion.ListConceptProperties(testCs, CodeSystem.CONCEPTPROPERTY_NOT_SELECTABLE).Any());
         }
 
-
-        private void testService(ITerminologyService svc)
-        {
-            var vsUrl = "http://hl7.org/fhir/ValueSet/data-absent-reason";
-#pragma warning disable CS0618 // Type or member is obsolete
-            var result = svc.ValidateCode(vsUrl, code: "NaN", system: "http://hl7.org/fhir/data-absent-reason");
```

Comment:
Don't these tests have to be rewritten with `ValueSetValidateCode(Parameters, string, bool)` instead of being deleted?

Reply:
Yeah, you are right... Most of these tests were already ignored, but now I rewrote them